### DISCUSSION TITLE
Custom Blocks of predefined Items

### DISCUSSION THREAD

**Discussion by nesges** [JulianPrieber/littlelink-custom#782] [Feature requests]
I'm just setting up Linkstack for a TTRPG (tabletop roleplaying games) group and it would be nice to have a block that lets people show their favorite games, like Dungeons & Dragons, Shadowrun, Call of Cthulhu etc.

As admin of the site I'd like to be able to define a custom block, with a list of name+icon+url(+cssclass). The user would define a link of that block by choosing from the list of names.

This could be usefull not only for ttrpg people but virtualy any thematic community

**Comment by sanwecn** [comment]
Very supportive of this feature, hoping to add custom blocks to support simple HTML CSS JavaScript PHP code ❤

**Comment by JulianPrieber** [comment]
https://github.com/LinkStackOrg/LinkStack/commit/c1a47556f54e7ccaf7bbcacd2d685e1a86653bbb
